plugins/outputs/graylog/graylog.go

```diff
@@ -84,10 +122,19 @@ func (g *Gelf) Write(message []byte) (n int, err error) {
 
 	n = len(message)
 
-	return
```

Comment:
why wasn't this returning anything before?

Reply:
It was naked return, ie. named return values `n` and `error` were returned. However lint-pr-changes check no longer allows naked returns.